clients/drcachesim/tools/schedule_stats.cpp

```diff
@@ -223,8 +223,12 @@ schedule_stats_t::parallel_shard_memref(void *shard_data, const memref_t &memref
     if (memref.instr.tid != INVALID_THREAD_ID)
         shard->counters.threads.insert(memref.instr.tid);
     if (memref.marker.type == TRACE_TYPE_MARKER) {
-        if (memref.marker.marker_type == TRACE_MARKER_TYPE_SYSCALL)
+        if (memref.marker.marker_type == TRACE_MARKER_TYPE_SYSCALL) {
             ++shard->counters.syscalls;
+            // The scheduler will block on a long-enough-latency non-marked
+            // syscall, so we label all to get the right % voluntary.
```

Comment:
Document that maybe_blocking numbers will be an overestimation.

We mark all system calls as blocking, will that number of useful?

Reply:
The counter is separate and we are not over-counting.  This saw_ boolean is only used for voluntary.

Reply:
Renamed to saw_syscall to avoid confusion.